### PR TITLE
Add Samsung Galaxy J5 2016 (SM-J510MN)

### PR DESCRIPTION
**Pull request by Ygor6889**

### CODE REVIEW

File: dts/msm8916/msm8916-samsung-r04.dts [hunk 1]
```diff
@@ -29,9 +29,20 @@
 		samsung,muic-reset {
 			i2c-gpio-pins = <105 106>;
 			i2c-address = <0x25>;
-		};
-	};
+        	};
+        };
```
Comment (wonderfulShrineMaidenOfParadise) [inline dts/msm8916/msm8916-samsung-r04.dts:33]:
```suggestion
		};
	};
```

File: dts/msm8916/msm8916-samsung-r04.dts [hunk 2]
```diff
@@ -29,9 +29,20 @@
...
+	j5xnlteub {
+		model = "Samsung Galaxy J5 2016 (SM-J510MN)";
+		compatible = "samsung,j5xnlteub", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "J510MN*";
 
+		samsung,muic-reset {
+			i2c-gpio-pins = <105 106>;
+			i2c-address = <0x25>;
+                };
+        };
```
Comment (wonderfulShrineMaidenOfParadise) [inline dts/msm8916/msm8916-samsung-r04.dts:44]:
```suggestion
		};
	};       
```

**Review by wonderfulShrineMaidenOfParadise — COMMENTED** [review]
Please fix the indents with tabs and tailing white spaces.

**Comment by TravMurav** [comment]
Thanks!

**Pull request merged**